couchdb/10-docker-default.ini

```diff
@@ -10,6 +10,8 @@ os_process_limit = 1000
 os_process_timeout = 60000
 max_dbs_open = 5000
 attachment_stream_buffer_size = 16384
+users_db_security_editable = true
```

Comment:
Why do we need this?

Reply:
We have this ancient migration that adds `national_admin` to `_users`: https://github.com/medic/cht-core/blob/master/api/src/migrations/add-national_admin-role.js#L58
We have historically allowed `national_admins` to make changes to users, we would be breaking backwards compatibility.

Reply:
I think it's probably time to break that backwards compatibility. I'm worried that changing this setting puts us at a security risk, and the national_admin role has been deprecated for years.

What do you think about a migration to add the `_admin` role to all `national_admin`s?

This could be done as a separate PR.

Reply:
Hm, ok. I'm thinking about other ways we would break functionality by making all `national_admin`s actual admins. 
This issue comes to mind: https://github.com/medic/cht-core/issues/8243 where we would have previously allowed to turn off permissions for `national_admins`.